.changes/947.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "type": "internal",
+        "description": "resolve symlinks for cargo and xargo home",
+        "issues": [373]
+    },
+    {
+        "type": "fixed",
+        "description": "mount cargo, xargo, and the sysroot at the same path as on the host to avoid unnecessary recompilation.",
```

Comment:
this is only true for linux, no?

Reply:
Nope. Also true for Windows (tested) and macOS. It turns out, it doesn't need a recompile if the Windows-style path points to the same location as the WSL path.